README.md

```diff
@@ -90,4 +90,5 @@ This section describes the folder strcuture of this project:
 - `src/physics`: all source files relating to physical calculations
 - `src/simulation`: all source files relating to the `Simlation` class
 - `src/MolSim.cpp`: source file holding main function
+- `src/generators`: all source files related to generating initial particle clusters
```

Comment:
I would rather move this whole code to `models/generators`, as it is about generating the "models" (Particles) to be used. This would keep the directories in the `src` folder divided into the main components of the project :)